.github/workflows/testing.yml

```diff
@@ -63,3 +63,47 @@ jobs:
           COVERALLS_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
           make coveralls
+  run-tests-ee:
+    if: github.event_name == 'push' ||
+      github.event.pull_request.head.repo.full_name != github.repository
+
+    runs-on: ubuntu-latest
+
+    env:
+      TEST_TNT_EE: 1
+
+    strategy:
+      fail-fast: false
+      matrix:
+        sdk-version: ["2.10.0-beta2-59-gc51e2ba67-r469"]
+
+    steps:
+      - name: Clone the connector
+        uses: actions/checkout@v2
+
+      - name: Setup Tarantool ${{ matrix.sdk-version }}
+        run: |
+          ARCHIVE_NAME=tarantool-enterprise-bundle-${{ matrix.sdk-version }}.tar.gz
+          curl -O -L https://${{ secrets.SDK_DOWNLOAD_TOKEN }}@download.tarantool.io/enterprise/${ARCHIVE_NAME}
+          tar -xzf ${ARCHIVE_NAME}
+          rm -f ${ARCHIVE_NAME}
+          sudo cp tarantool-enterprise/tarantool /usr/bin/tarantool
+          sudo cp tarantool-enterprise/tarantoolctl /usr/bin/tarantoolctl
+
+      - name: Setup golang for the connector and tests
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.13
+
+      - name: Install test dependencies
+        run: make deps
+
+      - name: Run tests
+        run: make test
+
+      - name: Run tests, collect code coverage data and send to Coveralls
+        if: ${{ matrix.coveralls }}
+        env:
+          COVERALLS_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          make coveralls
```

Comment:
With your changes, we have two jobs for code coverage. I suppose Coveralls will show a coverage for a job (Community or EE) that post data last. It is not good. In ideal world, we need to measure code coverage for both editions and somehow show both separately on Coveralls.

Reply:
Thank you. Right now we have only one go-tarantool build: for CE and EE. EE test = CE test + some tests. I see three ways:

1. The result with EE, we assume that it is the most complete.
2. Merge all results.
3. Separate result for each build (Tarantool CE 1.10, Tarantool CE 2.8, Tarantool CE 2.9, Tarantool EE 2.10 etc)

I prefer 2 (if possible) or 1. What do you think?

Reply:
I chose the first one because merging is not supported by goveralls.
(but I can search for alternatives)

Reply:
I'm ok with the first approach.

Reply:
(Resolving the discussion so.)